backend/app/services/channels/telegram/service.py

```
@@ -0,0 +1,651 @@
+            if response:
+                # Edit the original message with the response
+                try:
+                    await update.callback_query.message.edit_text(response)
+                except Exception:
+                    # If edit fails, send new message
+                    await update.callback_query.message.reply_text(response)
+        finally:
+            db.close()
+
+    async def _handle_message(
+        self, update: Update, context: ContextTypes.DEFAULT_TYPE
```

Ruff (0.14.14)

[warning] Unused method argument: `context`

(ARG002)